masquerade-core/src/main/java/com/flipkart/masquerade/util/FallbackSpecification.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Flipkart Internet, pvt ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.flipkart.masquerade.util;
+
+/**
+ * Created by shrey.garg on 18/07/17.
+ */
+public class FallbackSpecification {
```

Comment:
Instead of a spec to a static method in a class, can't this be an interface (say Fallback) which has one method to take an object and return a String? Through configuration, apps can pass a reference to an implementation of this interface which can be used in the generated cloak.

Advantage would be to make the signature/contract of fallback explicit (that it has to take ab object and return a string). Later we might even change, to pass eval etc.